modules/ArchiveLibrary/Folder/Repositories/FolderRepository.php

```diff
@@ -189,6 +189,9 @@ public function getFoldersAndFilesByParent(
             if ($parentId != null) {
                 $foldersQuery->where('parent_id', $parentId);
             }
+            else{
+                $foldersQuery->whereNull('parent_id');
```

Comment:
Consider adding a space after `whereNull` for better readability:

```suggestion
                $foldersQuery->whereNull('parent_id');
```